client/views/chat.tpl

```diff
@@ -12,7 +12,7 @@
 		<span title="{{topic}}" class="topic">{{{parse topic}}}</span>
 	</div>
 	<div class="chat">
-		<div class="show-more {{#equal messages.length 100}}show{{/equal}}">
+		<div class="show-more{{#if messages.length}} show{{/if}}">
```

Comment:
What does that mean really? Show the button if the channel has messages?!

Reply:
Yep, because it's gonna send 1 message per channel, and 100 just for the active.

Reply:
I see, thanks!